src/leo-client-app/src/components/logModal.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
```

Comment:
import only what you need not everything

Reply:
fixed